cmd/registry/metrics/metrics.go

```diff
@@ -0,0 +1,51 @@
+package metrics
+
+import (
+	"github.com/apigee/registry/rpc"
+)
+
+// ComputeMetrics will compute the metrics proto for a list of Classified Diffs.
+func ComputeMetrics(calculatedDiffs []*rpc.ClassifiedChanges) *rpc.Metrics {
+	stats := calculateStats(calculatedDiffs)
+	return caluclateMetrics(stats)
+}
```

Comment:
Consider splitting this into two functions, `ComputeStats(...*rpc.ClassifiedChanges) *rpc.Stats` and `ComputeMetrics(*rpc.Stats) *rpc.Metrics`. This would also simplify the API by removing the need to include `Stats` as a submessage of `Metrics`.

Note: The `...` in `ComputeStats` makes it a [variadic function](https://gobyexample.com/variadic-functions) that can be passed any number of individual args, or a slice using the spread operator. It's just a convenience for users of the package.

Reply:
Done